crates/command_silly/src/lib.rs

```diff
@@ -227,6 +227,21 @@ silly_command! {
     }
 }
 
+silly_command! {
+    /// :3
+    fn rickroll("the victim") {
```

Comment:
New commands should be in Early Access! (Since there are five Early Access commands already, you should remove the oldest one from Early Access to keep it at five.)
```suggestion
    #[early_access]
    fn rickroll("the victim") {
```